packages/kit/test/utils.js

```diff
@@ -97,14 +97,24 @@ export const test = base.extend({
 		// automatically wait for kit started event after navigation functions if js is enabled
 		const page_navigation_functions = ['goto', 'goBack', 'reload'];
 		page_navigation_functions.forEach((fn) => {
+			// @ts-ignore
```

Comment:
why is this needed now when it wasn't before?

Reply:
not sure but not `pnpm lint` fails without it.